From: Zhi Zhang <zhang.david2011@gmail.com>
Date: Tue, 8 Nov 2016 14:35:59 +0100
Subject: ceph: fix printing wrong return variable in ceph_direct_read_write()
Git-commit: a380a031cbe4323a3f638a8468c862510ace1919
Patch-mainline: v4.10-rc1
References: FATE#322288

Fix printing wrong return variable for invalidate_inode_pages2_range in
ceph_direct_read_write().

Signed-off-by: Zhi Zhang <zhang.david2011@gmail.com>
Signed-off-by: Ilya Dryomov <idryomov@gmail.com>
Acked-by: Luis Henriques <lhenriques@suse.com>
---
 fs/ceph/file.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/fs/ceph/file.c
+++ b/fs/ceph/file.c
@@ -906,7 +906,7 @@ ceph_direct_read_write(struct kiocb *ioc
 					pos >> PAGE_CACHE_SHIFT,
 					(pos + count) >> PAGE_CACHE_SHIFT);
 		if (ret2 < 0)
-			dout("invalidate_inode_pages2_range returned %d\n", ret);
+			dout("invalidate_inode_pages2_range returned %d\n", ret2);
 
 		flags = CEPH_OSD_FLAG_ORDERSNAP |
 			CEPH_OSD_FLAG_ONDISK |
